src/bors/command/parser.rs

```diff
@@ -30,19 +30,19 @@ impl CommandParser {
         // The order of the parsers in the vector is important
         let parsers: Vec<fn(Tokenizer) -> ParseResult> =
             vec![parser_ping, parser_try_cancel, parser_try];
-
+    
         text.lines()
-            .filter_map(|line| match line.find(&self.prefix) {
-                Some(index) => {
-                    let command = &line[index + self.prefix.len()..];
+            .filter_map(|line| {
+                if let Some((command, kv_pairs)) = parse_key_value(line) {
```

Comment:
We still need to parse the bot prefix first, as before. Let's stick to the following command syntax for now:
```
@<prefix> <command> <key1>=<val1> <key2>=<val2>
@bors try
@bors try parent=foo
@bors try job=msvc parent=foo opts=debug,assertions
```
So first we need to look for the bot prefix (same as before), and if we find it, then try to parse a command and a list of key-value pairs after it.